src/pages/Settings/EmailSettings.vue

```diff
@@ -645,6 +648,7 @@ export default {
 				}
 				this.$showTipMsg('Your email settings have been saved');
 			} catch (err) {
+				console.log(err);
```

Comment:
Could you please use the logFormatter for this one?

Reply:
👍